spire/templates/apps/dovetail-spots.yml

```diff
@@ -313,6 +313,12 @@ Resources:
             - Name: SPOTS_HOST
               Value: !Ref SpotsHostname
           Essential: true
+          HealthCheck:
+            Command: [ "CMD-SHELL", !Sub "curl --fail http://localhost:${kWebApplicationPort}/up || exit 1" ]
```

Comment:
Isn't the `|| exit 1` redundant?  The `--fail` is already returning non-0 if we've gotten to the `||`.